### PR TITLE
Gather btrfs-debug-tree info in case of RO filesystem hit

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: lib/console_yasttest.pm [hunk 1]
```diff
@@ -17,6 +17,14 @@ sub post_fail_hook() {
     # only upload if save_y2log succeeded
     if (!script_run "save_y2logs $fn") {
         upload_logs $fn;
+    } else {
+        # there is a severe problem, e.g. could be bsc#985850 or bsc#990384 so
+        # save more, let's hope there is enough memory for intermediate
+        # storage
+        # TODO just assuming that '/dev/vda2' is the root device here
+        $fn = '/dev/shm/vda2_brfs_debug_tree';
+        assert_script_run "btrfs-debug-tree /dev/vda2 &> $fn";
```
Comment (coolo) [inline lib/console_yasttest.pm]: you can use /dev/root for that - and tidy

Comment (okurz) [inline lib/console_yasttest.pm]: done

**Comment by okurz** [comment]
wait, *not* done

**Comment by okurz** [comment]
*now* it's done

**Comment by coolo** [comment]
let's see if we can catch it in real life

File: lib/console_yasttest.pm (hunk 1 above)
Comment (okurz) [inline lib/console_yasttest.pm]: did not work, see https://openqa.suse.de/tests/492117/file/yast2_i-root_brfs_debug_tree

Comment (okurz) [inline lib/console_yasttest.pm]: pushed the tiny change to master and want to trigger some tests on osd for the night :-)

Comment (coolo) [inline lib/console_yasttest.pm]: this is very strange. /dev/root is just a symlink

**Pull request merged**